src/transport_picoquic.cpp

```diff
@@ -810,12 +799,24 @@ PicoQuicTransport::enqueue(const TransportContextId& context_id,
             metrics.enqueued_objs++;
             stream_cnx->second.tx_data->push(bytes, ttl_ms, priority);
 
+            if (!stream_id) {
+                picoquic_runner_queue.push([=, this]() {
+                    picoquic_mark_datagram_ready(stream_cnx->second.cnx, 1);
+                });
+
+            } else {
+                picoquic_runner_queue.push([=, this]() {
+                    picoquic_mark_active_stream(
+                      stream_cnx->second.cnx, stream_id, 1, static_cast<StreamContext*>(&stream_cnx->second));
+                });
+            }
         } else {
             std::cerr << "enqueue dropped due to invalid stream: " << stream_id << std::endl;
             return TransportError::InvalidStreamId;
         }
     } else {
-        std::cerr << "enqueue dropped due to invalid contextId: " << context_id << std::endl;
+        //std::cerr << "enqueue dropped due to invalid contextId: " << context_id << std::endl;
+        //TODO: Add metrics to report invalid context
```

Comment:
This has been spotted a lot recently, maybe we could keep this log around? Unless it isn't helpful, then never mind.

Reply:
Maybe it would be better to add it as a debug instead. When it does happen, it happens a lot and that becomes a lot of noise.  Normally we shouldn't see this though.